### PR TITLE
Fixing rspecs for bet and bet_option

### PR DESCRIPTION
**Pull request by ghost**
Added required fields to test variables that are now validated upon creation, which caused them to break the rspec.

### CODE REVIEW

File: spec/models/bet_spec.rb [hunk 1]
```diff
@@ -11,11 +11,14 @@
     User.create(username: 'Jack', email: 'jack@mail.com', password: 'notjack')
   end
 
-  let(:bet1) do
-    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!')
+  let!(:bet1) do
+    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!',
+               expires_at: DateTime.current + 1, creator_id: bob.id)
```
Comment (qr8r) [inline spec/models/bet_spec.rb]: Rails has some fun helpers for playing with dates 

http://api.rubyonrails.org/classes/Date.html

File: spec/models/bet_spec.rb [hunk 2]
```diff
@@ -11,11 +11,14 @@
...
+  end
+  let!(:option1) { BetOption.create(option_text: 'Atlanta wins', bet: bet1) }
```
Comment (qr8r) [inline spec/models/bet_spec.rb]: I dont think these need to be bang methods.... Can you explain why you think you need them?

File: spec/models/bet_option_spec.rb [hunk 3]
```diff
@@ -11,8 +11,9 @@
     User.create(username: 'Jack', email: 'jack@mail.com', password: 'notjack')
   end
 
-  let(:bet1) do
-    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!')
+  let!(:bet1) do
+    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!',
+               expires_at: DateTime.current + 1, creator_id: bob.id)
```
Comment (qr8r) [inline spec/models/bet_option_spec.rb]: Same comments as your previous test

**Review by qr8r — CHANGES_REQUESTED** [review]

**Comment by cbrunsdon** [comment]
jenkins test this please

**Review by qr8r — APPROVED** [review]

**Pull request merged**